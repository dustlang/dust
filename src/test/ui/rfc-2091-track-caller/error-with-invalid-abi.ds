#[track_caller]
extern "C" fn f() {}
//~^^ ERROR `#[track_caller]` requires Dust ABI

extern "C" {
    #[track_caller]
    fn g();
    //~^^ ERROR `#[track_caller]` requires Dust ABI
}

fn main() {}
