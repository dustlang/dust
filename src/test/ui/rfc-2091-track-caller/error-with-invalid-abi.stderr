error[E0737]: `#[track_caller]` requires Dust ABI
  --> $DIR/error-with-invalid-abi.rs:1:1
   |
LL | #[track_caller]
   | ^^^^^^^^^^^^^^^

error[E0737]: `#[track_caller]` requires Dust ABI
  --> $DIR/error-with-invalid-abi.rs:6:5
   |
LL |     #[track_caller]
   |     ^^^^^^^^^^^^^^^

error: aborting due to 2 previous errors

For more information about this error, try `dustc --explain E0737`.
